src/Tizen.NUI/src/internal/WebView/WebView.cs

```diff
@@ -1788,6 +1802,37 @@ public void RegisterGeolocationPermissionCallback(GeolocationPermissionCallback
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Creates a new hit test.
+        /// <param name="x">the horizontal position to query</param>
+        /// <param name="y">the vertical position to query</param>
+        /// <param name="mode">the mode of hit test</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public WebHitTest CreateHitTest(int x, int y, WebHitTest.HitTestMode mode)
```

Comment:
Like I mentioned above, I think it is better just doing "HitTest".
```C#
public WebHitTestResult HitTest(int x, int y, HitTestMode mode)
{
}
```

Reply:
Done.